xdg-ninja.sh

```diff
@@ -5,22 +5,20 @@ has_command() {
     return $?
 }
 
-USE_GLOW=false
-USE_BAT=false
-USE_PYGMENTIZE=false
-USE_HIGHLIGHT=false
+DECODER="cat"
```

Comment:
Why use `cat`? Maybe, just don't pipe, if there is no decoder

Reply:
I considered that, though to not pipe would require adding in an if statement which makes this much less elegant.

Reply:
Writing `DECODER="cat"` instead of `DECODER=` is much less elegant. Also, spawning external processes for no reason is slow

Reply:
You can keep it as is using cat.